docs/MAINLINE_STATE_2026-05-27.md

```
@@ -0,0 +1,161 @@
+
+Verify locally:
+
+```bash
+git fetch origin && git rev-parse HEAD && git rev-parse --short HEAD
+```
+
+## 2. Commands cited (2026-05-27)
+
+### `git rev-parse HEAD`
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)